gin/converter.h:363:24: error: no member named 'FromV8' in 'gin::Converter<unsigned long>'

Index: gin/converter.h
--- gin/converter.h.orig
+++ gin/converter.h
@@ -102,6 +102,16 @@ struct GIN_EXPORT Converter<uint64_t> {
 };
 
 template<>
+struct GIN_EXPORT Converter<unsigned long> {
+  // Warning: JavaScript cannot represent 64 integers precisely.
+  static v8::Local<v8::Value> ToV8(v8::Isolate* isolate,
+                                    unsigned long val);
+  static bool FromV8(v8::Isolate* isolate,
+                     v8::Local<v8::Value> val,
+                     unsigned long* out);
+};
+
+template<>
 struct GIN_EXPORT Converter<float> {
   static v8::Local<v8::Value> ToV8(v8::Isolate* isolate,
                                     float val);
